src/officehours_api/admin.py

```diff
@@ -1,17 +1,78 @@
+import logging
+
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
 from django.contrib.auth.models import User
+from django.http import HttpResponse
 from safedelete.admin import SafeDeleteAdmin, highlight_deleted
 
 from officehours_api.models import Queue, Meeting, Attendee, Profile
+from officehours_api.views import ExportMeetingStartLogs
+
+logger = logging.getLogger(__name__)
+
+admin.site.site_title = admin.site.site_header = 'ROHQ Admin'
+admin.site.index_title = 'Home'
+
+
+class ExporterAdminMixin:
```

Comment:
It felt to me like this class is the Python "mixin" style.  Maybe I'm using the term incorrectly.  If this doesn't seem like a "mixin", tell me and I will rename the class.